department-of-veterans-affairs/va.gov-cms
# VACMS-12705: enable translation modules and add Spanish and Tagalog languages

## Description

Fulfills #12705 

## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

As user with administrator privileges:
1. Verify that Spanish and Tagalog are listed as available language: https://pr12890-tg7jnz6wknbionjxjvbsgwei7tpx0wwb.ci.cms.va.gov/admin/config/regional/language
2. Verify that no entity types are enabled as translatable: https://pr12890-tg7jnz6wknbionjxjvbsgwei7tpx0wwb.ci.cms.va.gov/admin/config/regional/content-language (2 will be checked, but also display as 'not translatable')
3. Visit a Content Type field form configuration page and verify that the 'Language' field is hidden: https://pr12890-tg7jnz6wknbionjxjvbsgwei7tpx0wwb.ci.cms.va.gov/admin/structure/types/manage/campaign_landing_page/form-display
4. Visit a Content Type field display configuration page and verify that the Language field is hidden: https://pr12890-tg7jnz6wknbionjxjvbsgwei7tpx0wwb.ci.cms.va.gov/admin/structure/types/manage/campaign_landing_page/display
5. Confirm that Configuration Translation admin can be reached: https://pr12890-tg7jnz6wknbionjxjvbsgwei7tpx0wwb.ci.cms.va.gov/admin/config/regional/config-translation
6. Confirm that Interface Translation admin can be reached: https://pr12890-tg7jnz6wknbionjxjvbsgwei7tpx0wwb.ci.cms.va.gov/admin/config/regional/translate

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
